### DISCUSSION TITLE
<security problem related for repository>

### DISCUSSION THREAD

**Discussion by DanningMa** [community/community#41665] [Code Security]
### Select Topic Area

Product Feedback

### Body

I found an open repository but the user who owns it has a mark: "This user is a registered cybercriminal. Learn about GitHub Terms of Service." Is this a hacker warning or mean the repository is dangerous and we should not download it?

**Comment by radar-blip** [comment]
Is this message located next to the users profile picture?

If that is the users status message (the little expandable ball next to the profile picture) I wouldnt worry to much about it.
Users can put just about anything they want as their status message.
Probably someone just being funny.

It will also display like below if user icon is hovered over from another page.

![statuscheck](https://user-images.githubusercontent.com/39196767/207483537-190214c7-c473-469d-92fc-e910eeeb45ba.png)

**Comment by winstxnhdw** [comment]
bruh... hahahaha
